cypress/support/step_definitions/buttonToggleGroupSteps.js

```diff
@@ -0,0 +1,85 @@
+import { helpIcon, labelPreview, labelHelpInput } from "../../locators/commonLocators";
+import {
+  inputWidthInput, inputWidthPreview, fieldHelpInput,
+  fieldHelpPreview, labelHelpPreview, lableInlineCheckbox, buttonToggleGroupPreview, labelAlignInput, labelWidthInput
+} from "../../locators/buttonToggleGroupLocators";
+
+Given('I open Button Toggle Group component page', () => {
+  cy.visit(Cypress.env('button_toggle_group_component_url'));
+});
+
+Then('Label on preview is {string}', (label) => {
+  labelPreview().should('have.text', label)
+});
+
+When('I set label help to {string}', (labelHelp) => {
+  labelHelpInput().clear().type(labelHelp)
+})
+
+When('I hover mouse on help icon', () => {
+  helpIcon().trigger('mouseover')
+});
+
+When('I set input width to {string}', (width) => {
+  inputWidthInput().clear().type(width)
+})
+
+Then('Input width is set to {string}', (width) => {
+  inputWidthPreview().should('have.attr', 'style').should('contain', `width: ${width}%`)
+})
+
+Then('Input width is not set', () => {
+  inputWidthPreview().should('not.have.attr', 'style')
+})
+
+When('I set field help to {string}', (width) => {
+  fieldHelpInput().clear().type(width)
+})
+
+Then('Field help on preview is set to {string}', (text) => {
+  fieldHelpPreview().should('have.text', text)
+})
+
+Then('Label help on preview is set to {string}', (text) => {
+  labelHelpPreview().should('have.text', text)
+})
+
+When('I check label inline checkbox', () => {
+  lableInlineCheckbox().check()
+})
+
+When('I uncheck label inline checkbox', () => {
+  lableInlineCheckbox().uncheck()
+})
+
+Then('Button Toggle Group component has label-inline property', () => {
+  buttonToggleGroupPreview().should('have.class', 'common-input--label-inline')
```

Comment:
We are working to remove classNames from our markup, should we be writing tests for our new components rather than the old ones?

Reply:
This test concerns Toogle Group Storybook Component that has been already developed and merged to master. If in future classNames will be removed then test will be updated with those changes

Reply:
I isolate Given steps to Background to reduce codelines